test/integration/content-sidebar/ContentSidebar.e2e.test.js

```diff
@@ -0,0 +1,30 @@
+// <reference types="Cypress" />
+
+describe('ContentSidebar', () => {
+    beforeEach(() => {
+        cy.visit('/ContentSidebar');
+    });
+
+    it('Toggling the skills button should hide the sidebar', () => {
+        // Sidebar should be open by default
+        cy.getByTestId('bcs-content').should('exist');
+        cy.getByTestId('sidebarskills').should('have.class', 'bcs-nav-btn-is-selected');
+
+        cy.getByTestId('sidebarskills').click();
+        cy.getByTestId('bcs-content').should('not.exist');
+
+        cy.getByTestId('sidebarskills').click();
+        cy.getByTestId('bcs-content').should('exist');
+    });
+
+    it('Clicking other tabs should switch the sidebar tab', () => {
+        // Sidebar should be open by default
+        cy.getByTestId('bcs-content').should('exist');
+        cy.getByTestId('sidebarskills').should('have.class', 'bcs-nav-btn-is-selected');
```

Comment:
Consider adding a sidebar-specific `data-testid` to the sidebar content component. That will allow you to avoid keying on specific CSS classes via something like `cy.getByTestId('sidebarskills-content').should('be.visible');`